tests/run-make/prefer-rlib/rmake.rs

```diff
@@ -0,0 +1,12 @@
+//@ ignore-cross-compile
+
+use run_make_support::{path, run, rust_lib_name, rustc};
+use std::fs::remove_file;
+
+fn main() {
+    rustc().input("bar.rs").crate_type("dylib").crate_type("rlib").run();
+    assert!(path(rust_lib_name("bar")).exists());
+    rustc().input("foo.rs").run();
+    remove_file(rust_lib_name("bar")).unwrap();
```

Comment:
Here we should also remove the dynamic library, to make sure that `foo` indeed compiled statically (which is the point of the test). If it didn't, it could still use `libbar.so`, and the test would succeed.

Reply:
Outch indeed!